woocommerce-services.php

```diff
@@ -357,14 +357,17 @@ public function load_textdomain() {
 		 * @codeCoverageIgnore
 		 */
 		public function init() {
+			$this->load_dependencies();
 			add_action( 'admin_init', array( $this, 'admin_enqueue_scripts' ) );
 
 			if ( ! WC_Connect_Options::get_option( 'tos_accepted', false ) ) {
 				add_action( 'admin_init', array( $this, 'admin_tos_notice' ) );
 				return;
+			} else if ( ! $this->check_jetpack_install() ) {
```

Comment:
Yes. It's either that, or the other way around (install Jetpack and then accept the ToS). I don't think we should show 2 notices at the same time, but I don't really care about the order :)